esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -51,6 +52,15 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
     const canDeschedule = workunit && workunit?.EventSchedule === 2;
     const canReschedule = workunit && workunit?.EventSchedule === 1;
 
+    const [DeleteConfirm, setShowDeleteConfirm] = useConfirm({
+        title: nlsHPCC.Delete,
+        message: nlsHPCC.YouAreAboutToDeleteThisWorkunit,
+        onSubmit: React.useCallback(() => {
+            workunit?.delete().catch(logger.error);
+            pushUrl("/workunits");
```

Comment:
This should probably be inside:
`workunit.delete().then(r=>pushUrl(...)).catch(`